src/models/topic.model.js

```diff
@@ -20,6 +20,12 @@ const topicSchema = new Schema(
                 ref: 'FlashCard',
             },
         ],
+        questionCard: [
+            {
+                type: Schema.Types.ObjectId,
+                ref: 'QuestionCard',
+            },
+        ],

```

Comment:
question card tương tự flash card nhé, lưu trong cards, bạn xử lý để lưu 1 trong 2 tùy TH nhé, tham khảo:
```suggestion
        cardType: {
          type: String,
          required: true,
          enum: ['FlashCard', 'QuestionCard']
        },
        cards: [
          {
            type: Schema.Types.ObjectId,
            refPath: 'cardType'
          }
        ]
```